.changeset/cuddly-radios-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lowdefy/actions-core': minor
+---
+
+Feature: Add new SetFocus Action
```

Comment:
Please remove "Feature:" as this will be generated automatically, and change to:

Add the SetFocus action that sets focus on a specified HTML element

Reply:
0d11ab656b390b068df51a96141ed45f23eef6a9